bucket/discord-delete.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.3.1",
+    "description": "Tool to delete Discord message history",
```

Comment:
> Be warned that using this tool could result in the termination of your account,

I do not like expose this tool to scoop users.

Reply:
No comment. It's just how the tool works. I see no reason why this should be excluded from the bucket.

Reply:
/unsubscribe

Allowing users to install the tool, which will terminate their accounts without prior notice is dangerous and does not make sense.

I will not be the one who will merge this and be responsible for this.

Reply:
That's not what it says. It says that it *could* result in termination, and to date, this has never happened to a user of the tool.
It's a disclaimer to warn that it is not risk-free.